src/containers/org/OrgContainer.js

```diff
@@ -78,6 +80,9 @@ const OrgContainer = ({
   const isInstalled :boolean = useSelector(selectIsAppInstalled(APPS.ACCESS_REQUESTS, organizationId));
   const collaborationsByOrganizationId = useSelector(selectCollaborationsByOrgId(organizationId));
 
+  const collaborationHref = ORG_COLLABORATIONS.replace(':organizationId', organizationId);
+  const orgHref = ORG.replace(':organizationId', organizationId);
```

Comment:
Please use `ORG_ID_PARAM` in the routes file.

```suggestion
  const orgHref = ORG.replace(ORG_ID_PARAM, organizationId);
```